hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -1085,6 +1085,19 @@ private int getWhenNoSSLDetected() {
     public static final HazelcastProperty MOBY_NAMING_ENABLED
             = new HazelcastProperty("hazelcast.member.naming.moby.enabled", true);
 
+    /**
+     * Client protocol frame size limit (in bytes) for unverified connections (i.e. maximal length of authentication message
+     * frame).
+     */
+    public static final HazelcastProperty CLIENT_PROTOCOL_UNVERIFIED_FRAME_LENGTH =
+            new HazelcastProperty("hazelcast.client.protocol.max.frame.length", 1024);
```

Comment:
Maybe include the word bytes in the prop name? We tend to do that with sizes/lengths on properties, easier to know what to put as a value without looking it up every time.